docs/podman-container-exists.1.md

```diff
@@ -0,0 +1,45 @@
+% PODMAN(1) Podman Man Pages
+% Brent Baude
+% November 2018
+# NAME
+podman-container-exists- Check if a container exists in local storage
+
+# SYNOPSIS
+**podman container exists**
+[**-h**|**--help**]
+[**-q**|**--quiet**]
+CONTAINER
+
+# DESCRIPTION
+**podman container exists** checks if a container exists in local storage. You may use the **ID** or **Name**
```

Comment:
I've been trying to keep "you" out of man pages.  Can this be changed to something like:  "Either the **ID** or the **Name** of the container can be used as input."

Reply:
passive voice it is